bitcoin_utils.py

```diff
@@ -0,0 +1,75 @@
+import os
+import requests
+from bitcoin import SelectParams
+from bitcoin.wallet import CBitcoinSecret, P2PKHBitcoinAddress
+from bitcoin.core import b2lx, lx
+from bitcoin.core.script import CScript
+from bitcoin.core.scripteval import VerifyScript, SCRIPT_VERIFY_P2SH
+from bitcoin.rpc import Proxy
+
+
+def create_wallet():
+    secret = CBitcoinSecret.from_secret_bytes(os.urandom(32))
+    address = P2PKHBitcoinAddress.from_pubkey(secret.pub)
+    return str(address), str(secret)
+
+
+def get_balance(address):
+    url = f'https://blockchain.info/q/addressbalance/{address}?confirmations=6'
+    response = requests.get(url)
+    if response.status_code == 200:
+        return int(response.text) / 100000000  # Convert from satoshis to BTC
+    else:
+        raise Exception(f'Error fetching balance: {response.text}')
+
+
+def send_bitcoin(sender_secret, recipient_address, amount_btc, fee_btc):
+    SelectParams('mainnet')
+    rpc = Proxy()
+
+    sender_secret = CBitcoinSecret(sender_secret)
+    sender_address = P2PKHBitcoinAddress.from_pubkey(sender_secret.pub)
+    recipient_address = P2PKHBitcoinAddress(recipient_address)
+
+    amount_satoshis = int(amount_btc * 100000000)
+    fee_satoshis = int(fee_btc * 100000000)
+
+    unspent = [utxo for utxo in rpc.listunspent(0) if utxo['address'] == sender_address]
+    if not unspent:
+        raise Exception('No available UTXOs for the sender address.')
+
+    unspent.sort(key=lambda x: -x['amount'])
```

Comment:
you should randomize the list, if you sort by amount you'll either grind your utxos to dust or pay way too much fees